src/pages/hosts/HostsTable.tsx

```diff
@@ -301,3 +303,6 @@ export const HostsTable: React.VFC<Props> = ({
 };
 
 const rowKey = ({ id }: { id: string }): string => id;
+const Owner = styled.span`
+  word-break: break-all;
```

Comment:
You can reuse the word break style we already have
https://github.com/evergreen-ci/spruce/blob/620562260a9e06b7996671a45e9ab33f9f2399cf/src/components/styles/Typography.tsx#L11